portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,122 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+* Plays a specified melody
+ */
+function play(melody) {
```

Comment:
You are using both "function" keywords and arrow functions. Can you be consistent over your codebase? (Choose one -- maybe arrow functions with 'const'):
https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions

Reply:
Done!